Week-2/1.Flight-booking.js

```diff
@@ -0,0 +1,21 @@
+"use strict";
+
+function getFullName(firstname, surname, useFormalName, gender) {
+  firstname = firstname.trim();
+  surname = surname.trim();
+
+  if (!firstname || !surname) {
+    return "Invalid Input";
+  }
+
+  const fullName = `${firstname} ${surname}`;
+
+  if (useFormalName === true) {
+    const type = gender === "male" ? "Lord" : "Lady";
+    return `${type} ${fullName}`;
```

Comment:
yown i'm happy with this na

if it's me though, instead of returning here, I'd just do fullName = \`${type} ${fullName}\` para isa lang yung return, but this is ok na

Reply:
yeah sige will keep this in mind